rfcs/0000-c-unwind-and-project-group-charter.md

```diff
@@ -6,146 +6,152 @@
 # Summary
 [summary]: #summary
 
-* First, to modify Rust functions with the "C" abi to abort on unwinding, as proposed
-  in [rust-lang/rust#52652].
-* Second, to introduce the "C unwind" ABI, with virtually all details
-  (but not quite all) left as "to be determined".
-    * The **intent** of this ABI, however, is to permit Rust panics to
-      be "translated" into a "native" unwinding mechanism, though
-      precisely which is dependent on the target platform or other
-      compiler options.
-        * (In practice, this translation is a no-op, since Rust panics
-          use the native mechanism, but this is explicitly not
-          required for the future.)
-    * Folks currently using the "C" ABI to do unwinding can migrate to this new
-      "C unwind" ABI, which captures their intentions. Their code is no more stable
-      than it was, but they are on the right path.
-* Finally, to create a "project group" with the purpose of designing subsequent
-  RFCs to flesh out the details of the "C unwind" ABI
-    * The "project group" term is newly introduced, but it corresponds
-      to a kind of working group -- one with the goal of fleshing out
-      a particular proposal or completing a project.
-    * We aim to specify how "C unwind" works on major platforms
-    * Goal is to enable Rust panics to propagate across native frames
-    * And perhaps to enable native exceptions to propagate across Rust frames
-    * But not to allow catching or throwing native exceptions from Rust code
+* Rust functions with the "C" abi will (abort on unwinding)[rust-lang/rust#52652].
+* The "C unwind" ABI will be introduced with unspecified behavior but the
+  explicit intention to permit Rust panics to propogate through native frames
+* Announce "project group" with the purpose of designing subsequent RFCs to
+  flesh out the details of the "C unwind" ABI; invite interested Rustaceans to
+  join
 
 # Motivation
 [motivation]: #motivation
 
-## We have no FFI ABI that permits unwinding, and we need one
+## We have no FFI ABI that permits unwinding, and we need one.
 
 Rust has long held that the `extern "C"` ABI does not permit
 unwinding. Therefore, attempts to unwind through a `extern "C"`
 barrier are considered [Undefined Behavior]. However, in practice, a
-number of projects have found that "things work fine" so long as
-certain conditions are met (for example, the frames that are to be
-unwound do not contain destructors).
+number of projects have found that "things work fine" in certain cases, such as
+when unwinding over C code compiled with Clang using the `-fexceptions`
+compiler option.
+
+> XXX I don't believe we've ever seen improper destructor behavior when
```

Comment:
Throughout, I've used `> XXX` to indicate a comment from me. You had also used `XXX` to mark something like that, so I changed it to `> Niko XXX`.